### PR TITLE
Restrict editing of published nodes.

### PR DESCRIPTION
**Pull request by katzmo**
Provides extra permissions to edit or delete published nodes.

### CODE REVIEW

File: change_publishing_status_permission.module [hunk 1]
```diff
@@ -32,6 +40,22 @@ function change_publishing_status_permission_form_node_form_alter(&$form, &$form
...
 }
 
+/**
+ * Implements hook_node_access().
+ *
+ * Restricts editing and deleting of published nodes.
+ */
+function change_publishing_status_permission_node_access($node, $op, $account) {
+  $restricted_ops = ['update', 'delete'];
+  if (in_array($op, $restricted_ops) &&
```
Comment (torotil) [inline change_publishing_status_permission.module]: Please don’t wrap conditions: https://www.drupal.org/docs/develop/standards/coding-standards#linelength

**Review by torotil — CHANGES_REQUESTED** [review]
Minor code-style issue. Otherwise fine. ;-)

File: change_publishing_status_permission.module (hunk 1 above)
Comment (katzmo) [reply change_publishing_status_permission.module]: So instead have a very long line with conditions? Or spilt it over multiple ifs?

**Review by torotil — APPROVED** [review]
perfect!

**Pull request merged**